content/en/docs/topics/kubernetes_apis.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kubernetes APIs"
+description: "Explains Kubernetes APIs in Helm templates"
+aliases: ["docs/k8s_apis/"]
+---
+
+Helm templates specify a [Kubernetes API group](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups) when defining a Kubernetes object, similar to a Kubernetes manifest file. It is specified in the `apiVersion` field of the template and it identifies the API version of the object.
+
+Like all API-driven system, Kubernetes has a [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/) for its APIs. This means that after a certain number of releases, an API version cited for deprecation will no longer work in a Kubernetes version.
+
+Helm users and chart maintainers need to be aware of Kubernetes APIs cited for deprecation to avoid issues. 
+
+For chart maintainers, you should update any Kubernetes API versions cited for deprecation to their new supported equivalents. This means keeping aware of upcoming Kubernetes releases and any updates on deprecated API for that release. This is an example of a [blog](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) which provides information on the deprecated APIs for Kubernetes 1.16.
+
+For Helm users, you should audit the charts that you use and make sure that the APIs are supported. If not then you need to check for the latest version of the chart. Additionally, if you have any current charts deployed (Helm releases) with APIs due for deprecation, you need to upgrade the releases using charts with supported APIs before upgrading to the Kubernetes version which deprecates the API. If you don't upgrade these releases in advance, you will have an error similar to the following when trying to upgrade in the new Kubernetes version:
```

Comment:
1. What do you mean here by `make sure that the APIs are supported`?
Do you mean `non-deprecated APIs` or `non-removed APIs`?

1. `...with APIs due for deprecation...`
should maybe be
`...with deprecated APIs due for removal...`

1. `...before upgrading to the Kubernetes version which deprecates the API.`
should maybe be
`...before upgrading to the Kubernetes version which removes the API.`

1. `...when trying to upgrade in the new Kubernetes...`
could be more specific as
`...when trying to upgrade with Helm in the new Kubernetes...`
This is just to be very clear that it is not the upgrade of Kubernetes itself that will fail.

Reply:
Changed this around to bring better clarification